lighthouse-cli/chrome-finder.ts

```diff
@@ -133,3 +182,27 @@ function canAccess(file: string): Boolean {
     return false;
   }
 }
+
+function uniq(arr: Array<any>) {
+    return arr.sort().filter((item: any, pos: number, _this: any) =>
+      !pos || item != _this[pos - 1]
+    );
+}
+
+function findChromeExecutables(desktopPath: string): Array<string> {
+  let installations: Array<string> = [];
+  if (canAccess(desktopPath)) {
+    let execPaths = execSync(`grep -ER "${chromeExecRegex}" ${desktopPath} | awk -F '=' '{print $2}'`)
```

Comment:
can you add a comment on what sort of output this grep|awk line generates? that would help explain the need for the argumentsregex replace.